.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
```

Comment:
Could you please switch it to Github Actions? Just copy paste whole .github folder from here https://github.com/contributte/bare/tree/master/.github.